web/packages/teleport/src/Console/DocumentNodes/DocumentNodes.tsx

```diff
@@ -128,6 +128,7 @@ const Container = styled(Box)`
   display: flex;
   flex: 1;
   max-width: 1024px;
+  height: fit-content;
```

Comment:
Do you remember why [`overflow: hidden` was added to StyledTableWrapper](https://github.com/gravitational/teleport/commit/5a46f044b56#diff-56a64e0fa19fefcaa7f5fd1bb401ab3097a04b27a60e8a2b15fb942a460f3046R102)? If it's possible to get rid of it then I think long-term it might be a better fix than adding `height: fit-content` here.

Reply:
The reasoning was for all our different tables (paginated, searchable, simple, etc.) to have rounded corners. Wrapping all tables with it and having it round the corners with a `borderRadius` made it so we didn't have to always manually add a border radius for every table individually.